pkg/policy/policy.go

```diff
@@ -236,6 +238,14 @@ func (r *Policies) GetResultID(result scan.Result) string {
 	return id
 }
 
+func (r *Policies) HasSeverity(severity severity.Severity) bool {
```

Comment:
probably better to change the param name to `resultSeverity` to avoid conflict with package name
```suggestion
func (r *Policies) HasSeverity(resultSeverity severity.Severity severity.Severity) bool {
```

Reply:
done